packages/plugins/rubygem-smart_proxy_discovery/rubygem-smart_proxy_discovery.spec

```diff
@@ -71,19 +78,24 @@ mv %{buildroot}%{gem_instdir}/settings.d/discovery.yml.example \
 
 %files
 %dir %{gem_instdir}
+%config(noreplace) %attr(0640, root, foreman-proxy) %{foreman_proxy_settingsd_dir}/discovery.yml
```

Comment:
Would it make sense to use `%{plugin_name}` here as well as in the %install of it?

Reply:
probably yes

Reply:
so this is generated in a loop https://github.com/theforeman/foreman-packaging/blob/360dc68181bea3c3684a2dff713c4cd880dc360d/gem2rpm/smart_proxy_plugin.spec.erb#L130-L133

which makes it not easy to replace with `%{plugin_name}` (as there might be more configs)

Reply:
Hmm ok, fair enough.